apps/web/src/app/globals.css

```
@@ -211,6 +211,97 @@
+  /* Brand green (anchored to existing #4acf52 / #008755) */
+  --color-jersey:       #4acf52;
+  --color-jersey-deep:  #008755;
+  --color-jersey-bright: #22c55e;
+
+  /* Edges */
+  --color-paper-edge:   #d9d2bd;
+
+  /* ===== Redesign / Typography (Phase 0) ===== */
+
+  /* Font families — Freight already loaded via existing Adobe Typekit kit */
+  --font-display:     "freight-display-pro", Georgia, "Times New Roman", serif;
```

Stylelint (17.9.0)

[error] Expected "Georgia" to be "georgia" (value-keyword-case)

(value-keyword-case)